core/frontend/src/components/scanner/servicesScanner.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <span />
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import axios from 'axios'
+import { getModule } from 'vuex-module-decorators'
+import ServicesScannerStore from '@/store/servicesScanner'
+import {Service} from '@/types/SERVICE'
+
+const servicesHelper: ServicesScannerStore = getModule(ServicesScannerStore)
+
+/**
+ * Actual scanner for running services.
+ * This periodically fetches /helper/latest/web_services
+ * and updates the ServiceHelperStore with the available services
+ * @displayName Services Scanner
+ */
+export default Vue.extend({
+  name: 'ServicesFetcher',
+  data () {
+    return {
+      interval: 0,
```

Comment:
For someone reading the code, it may think that interval is the interval in time, and not the interval timer ID. 
Maybe: `interval_id`, `timer_id` or `request_timer_id` may help, also a small comment such as "We define the request timer ID by default as 0"

Reply:
done.